### PR TITLE
reworked the getting started section

### PR DESCRIPTION
**Pull request by fraincs**
- Initial update to documentation Getting Started Page.
- Removed dummy content.
- Added usage section.

### CODE REVIEW

File: apps/docs/content/tokens/getting-started/introduction.mdx [hunk 1]
```diff
@@ -8,41 +8,72 @@ order: 1
 
 At the heart of Workleap Design language are tokens, which drive its functionality. They are the key ingredients to everything UI. Workleap brand is dissected into design tokens, they live in the Hopper Design System.
 
+A design token can store any of the following elements:
+
+- A single value such as a `color`, a `font-size`, a `border-width`.
```
Comment (vicky-comeau) [inline apps/docs/content/tokens/getting-started/introduction.mdx:13]:
```suggestion
- A single value such as a `color`, `font-size`, or `border-width`.
```

File: apps/docs/content/tokens/getting-started/introduction.mdx [hunk 2]
```diff
@@ -8,41 +8,72 @@ order: 1
...
+- A collection of indivisible values such as a `box-shadow`.
+
+## Definitions
+
+Hopper aims to provide a three tier token system, each tiers have their specific purposes.
```
Comment (vicky-comeau) [inline apps/docs/content/tokens/getting-started/introduction.mdx:18]:
```suggestion
Hopper aims to provide a three tier token system, where each tier has their specific purpose.
```

File: apps/docs/content/tokens/getting-started/introduction.mdx [hunk 3]
```diff
@@ -8,41 +8,72 @@ order: 1
...
+
 <Card>
     <Image src="https://assets.workleap.com/hopper/webdoc/token-details.png" width="654" height="45" alt="Token tiers diagram" />
 </Card>
 
-## Definitions
+### Core tokens
+
+These tokens are referring to raw values such as an hexadecimal color code, a border-width, a font-size. They are the building blocks of the design system.
```
Comment (vicky-comeau) [inline apps/docs/content/tokens/getting-started/introduction.mdx:26]:
```suggestion
These tokens are referring to raw values such as a hexadecimal color code, border-width, or font-size. They are the building blocks of the design system.
```

We pronounce the Hs in English so it is just "a" :)

File: apps/docs/content/tokens/getting-started/introduction.mdx [hunk 4]
```diff
@@ -8,41 +8,72 @@ order: 1
...
+
+<Table category="color" data={[{
+    "name": "hop-sapphire-200",
+    "value": "#95b1ff"
+}]} />
+
+### Semantic tokens
+
+Adds the intention layer to tokens, they map to a core token, or tokens should we say as the semantic tier has a notion of themability. They are the tokens that are used in the components.
```
Comment (vicky-comeau) [inline apps/docs/content/tokens/getting-started/introduction.mdx:35]: This sentence is hard to decipher. Can you reword this please. Specifically after the last comma.

File: apps/docs/content/tokens/getting-started/introduction.mdx [hunk 5]
```diff
@@ -8,41 +8,72 @@ order: 1
...
+#### Dark
+
+<Table category="color" data={[{
+    "name": "hop-primary-surface-disabled",
+    "value": "#2040c7"
+}]} />
+
+### Component tokens
+
+Refers to the nuts and bolts of the components and their implementation details, they are the tokens exposed by components. They are tied to their respective component. _Note: As no components are currently available these tokens are not yet available._
```
Comment (vicky-comeau) [inline apps/docs/content/tokens/getting-started/introduction.mdx:53]: "they are the tokens exposed by components" should either be a new sentence of separated by a semi colon.

```suggestion
Refers to the nuts and bolts of the components and their implementation details; they are the tokens exposed by components. They are tied to their respective component. _Note: As no components are currently available, these tokens are not yet available._
```

File: apps/docs/content/tokens/getting-started/introduction.mdx [hunk 6]
```diff
@@ -8,41 +8,72 @@ order: 1
...
+<Table category="color" data={[{
+    "name": "hop-button-primary-disabled",
+    "value": "#2040c7"
+}]} />
+
+## When to use
+
+### For Developers
+
+Tokens should be used whenever Hopper components does not suit your current use case or if you can't use Hopper React components. They are also useful for situations where using a component does not make sense, such as adding a color for a background.
```
Comment (vicky-comeau) [inline apps/docs/content/tokens/getting-started/introduction.mdx:73]:
```suggestion
Tokens should be used whenever Hopper components do not suit your current use case or if you can't use Hopper React components. They are also useful for situations where using a component does not make sense, such as adding a color for a background.
```

File: apps/docs/content/tokens/getting-started/introduction.mdx (hunk 4 above)
Comment (fraincs) [reply apps/docs/content/tokens/getting-started/introduction.mdx:35]: Took it from design iirc, I'll update / remove.

File: apps/docs/content/tokens/getting-started/introduction.mdx [hunk 7]
```diff
@@ -8,41 +8,72 @@ order: 1
...
+
+It should be noted that if your current stack prevents you to directly use Hopper components it is recommended to use components tokens in priorty as they are mapped 1:1 with Hopper implementation.
```
Comment (vicky-comeau) [inline apps/docs/content/tokens/getting-started/introduction.mdx:75]:
```suggestion
It should be noted that if your current stack prevents you from directly using Hopper components, it is recommended to use component tokens in priority as they are mapped 1:1 with Hopper implementation.
```

File: apps/docs/content/tokens/getting-started/usage.mdx [hunk 8]
```diff
@@ -0,0 +1,47 @@
...
+
+In order to use Hopper fonts in your project, import the following:
+
+```css title="myfile.css"
+@import '@hopper-ui/tokens/fonts.css';
+```
+
+Font-face declarations are now imported in your project.
+
+Failing to do so would prevent you from using font families tokens such as `var(--hop-font-family-primary)`.
```
Comment (vicky-comeau) [inline apps/docs/content/tokens/getting-started/usage.mdx:43]:
```suggestion
Failing to do so would prevent you from using font family tokens such as `var(--hop-font-family-primary)`.
```

File: packages/tokens/src/style-dictionary/config.ts [hunk 9]
```diff
@@ -20,7 +20,7 @@ export const fontsConfig = {
                             "category": "asset",
                             "type": "font"
                         }
-                    },
+                    }
```
[the remark this answers is not in the document]
Comment (fraincs) [reply packages/tokens/src/style-dictionary/config.ts:23]: Will check

[the remark this answers is not in the document]
Comment (fraincs) [reply packages/tokens/src/style-dictionary/config.ts:23]: I was not.

File: apps/docs/content/tokens/getting-started/introduction.mdx [hunk 10]
```diff
@@ -8,41 +8,74 @@ order: 1
...
+<Table category="color" data={[{
+    "name": "hop-primary-surface-disabled",
+    "value": "#2040c7"
+}]} />
+
+### Component tokens
+
+Refers to the nuts and bolts of the components and their implementation details; they are the tokens exposed by components. They are tied to their respective component. These should not be used directly, unless you are building a component in a different stack than React.
+
+_Note: As no components are currently available these tokens are not yet available._
```
Comment (vicky-comeau) [inline apps/docs/content/tokens/getting-started/introduction.mdx:55]:
```suggestion
_Note: As no components are currently available, these tokens are not yet available._
```

**Review by franckgaudin — DISMISSED** [review]

**Pull request closed without merging**